engine/include/cubos/engine/plugins/tools/world_inspector.hpp

```diff
@@ -0,0 +1,21 @@
+#ifndef CUBOS_ENGINE_PLUGINS_WORLD_INSPECTOR_HPP
+#define CUBOS_ENGINE_PLUGINS_WORLD_INSPECTOR_HPP
+
+#include <cubos/engine/cubos.hpp>
+
+using namespace cubos::engine;
```

Comment:
Never use `using namespace` in header files - and in source files prefer `using`ing specifics types instead of the whole namespace. For example:

```cpp
// in a .cpp file
// instead of
using namespace cubos::engine;
// prefer
using cubos::engine::Cubos; // imports only that type into this namespace


```

Reply:
Don't forget to change the `using namespace` for `using X`